docs/man/telemprobd.1.rst

```diff
@@ -25,8 +25,8 @@ SYNOPSIS
 DESCRIPTION
 ===========
 
-The ``telemd`` program delivers locally generated telemetry records to a remote
-telemetry service. Telemetry data can be in any format, and is relayed as-is.
+The ``telemd`` program handles communication between telemetry client and telemetry
```

Comment:
s/telemd/telempostd

Looks like the search and replace should be done on the whole file.

Reply:
ack